includes/definitions/infinera-xtm.yaml

```diff
@@ -0,0 +1,32 @@
+os: infinera-xtm
+text: 'Infinera XTM'
+type: network
+icon: infinera
+mib_dir: infinera
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.8708
+
+poller_modules:
+    hr-mib: false
+    ipSystemStats: false
+    netstats: false
+    ospf: false
+    ucd-mib: false
+    ports: false
+    stp: false
+discovery_modules:
```

Comment:
Same

Reply:
Ok, will change that.

Only these four (hr-mib, ucd-mib, ports, stp) ?
The discovery modules are fine?

Reply:
Delete all. Including discovery. If they are not causing problems that is.